requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 pandas>=0.23
 pyarrow>=0.10
 numpy>=1.14
+pymysql
```

Comment:
is it just one function that we need? If that's the case, can we get rid of this dependency?